extensions/default/src/Panels/DataSourceSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classnames from 'classnames';
+import { useNavigate } from 'react-router-dom';
+
+import { Button } from '@ohif/ui';
+
+function DataSourceSelector() {
+  console.log('Rendering multi-data sources initial page');
+  const navigate = useNavigate();
+
+  const dsConfigs = window.config.dataSources;
```

Comment:
We shouldn't encourage people doing this. You can grab it from the extension manager